### ISSUE TITLE
Allow service healthchecks on different port

### ISSUE THREAD

**Issue by jonstacks** [aws/copilot-cli#3543]
## Details

It would be helpful to be able to configure the ALB target group to healthcheck the application on a different port than what requests go to. Some applications serve healthchecks or management APIs on a separate port for security concerns as the healthchecks might return internal information about the server such as the OS, etc. Being able to customize the [HealthCheckPort](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/target-group-health-checks.html#health-check-settings) would allow consumers to not have to implement IP address filtering or other mechanisms in application code while still being able to hit the healthchecks from a private network(assuming required security groups are in place) or by `exec`ing into the task.

## Manifest Changes

The proposed change would add a `http.healthcheck.port` key to the `manifest.yml` like so:

```yaml
# Your service name will be used in naming your resources like log groups, ECS services, etc.
name: frontend
type: Load Balanced Web Service

# Distribute traffic to your service.
http:
  path: '/'
  healthcheck:
    port: 8080  # This is the proposed change to allow healthchecking the target group on port 8080, instead of 80
    path: '/_healthcheck'
    success_codes: '200,301'

# Configuration for your containers and service.
image:
  build:
    dockerfile: ./frontend/Dockerfile
    context: ./frontend
  port: 80
```

I would be happy to contribute a patch for this if the change gets approved.

**Comment by huanjani** [comment]
Hello, @jonstacks! 
Thanks so much for opening this issue; your request makes a lot of sense. We currently allow non-default health check ports for NLBs (https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#nlb-healthcheck-port); it would be great to support this for ALBs, too. 
We would very much welcome your contribution if you're willing! Thank you!

**Comment by huanjani** [comment]
Your feature is now released in v1.19.0: https://github.com/aws/copilot-cli/releases/tag/v1.19.0! 🎉  Thanks so much for your contribution!
